otelfiber/config.go

```diff
@@ -28,6 +29,12 @@ func (o optionFunc) apply(c *config) {
 	o(c)
 }
 
+func Next(f func(ctx *fiber.Ctx) bool) Option {
```

Comment:
Not sure if to follow the pattern here should be WithNext, it's a bit different to the standard fibre middleware where we have ConfigDefault

Reply:
name it withNext, its a shortcut function to work with the defaults